cli/commands/pack.go

```diff
@@ -41,6 +48,79 @@ func init() {
 	packCmd.Flags().StringVar(
 		&ctx.Pack.StatboardUnitTemplatePath, "stateboard-unit-template", "", stateboardUnitTemplateUsage,
 	)
+
+	packCmd.Flags().StringSliceVar(&deps, "deps", []string{}, depsUsage)
+	packCmd.Flags().StringVar(&depsFile, "deps-file", "", depsFileUsage)
+}
+
+func parsePackageDeps() error {
+	var err error
+
+	if depsFile != "" && len(deps) != 0 {
+		return fmt.Errorf("You can't specify --deps and --deps-file flags at the same time")
+	}
+
+	if depsFile == "" && len(deps) == 0 {
+		defaultPackeDepsFilePath := filepath.Join(ctx.Project.Path, defaultPackageDepsFile)
+		if _, err := os.Stat(defaultPackeDepsFilePath); err != nil {
+			return fmt.Errorf("Failed to use default package dependencies file: %s", err)
+		}
+
+		depsFile = defaultPackeDepsFilePath
+	}
+
+	if depsFile != "" {
+		if _, err := os.Stat(depsFile); os.IsNotExist(err) {
+			return fmt.Errorf("Invalid path to file with dependencies: %s", err)
+		}
+
+		content, err := common.GetFileContent(depsFile)
+		if err != nil {
+			return fmt.Errorf("Failed to get file content: %s", err)
+		}
+
+		deps = strings.Split(content, "\n")
+	}
+
+	if !ctx.Tarantool.TarantoolIsEnterprise && (ctx.Pack.Type == pack.RpmType || ctx.Pack.Type == pack.DebType) {
```

Comment:
It's not about "parse".

What about 

```
fillDependencies(ctx *context.Ctx) {
  deps := parsePackageDeps(deps, depsFile)
  addTarantoolDepIfNeeded(ctx)
}
```

?

I'd like variables to be passed explicitly.